core/domain/state_domain.py

```diff
@@ -564,6 +620,14 @@ def get_all_html_content_strings(self):
         if self.solution:
             solution_html = self.solution.explanation.html
             html_list = html_list + [solution_html]
+            if self.solution.correct_answer:
+                interaction = (
+                    interaction_registry.Registry.get_interaction_by_id(
+                        self.id).to_dict())
```

Comment:
Any specific reason for convering this to dict?

Reply:
removed it